ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/facade/TestDefaultAuditor.java

```diff
@@ -109,6 +110,7 @@ public void testAuditFiltered()
         assertThat(timeMeasured).isLessThanOrEqualTo(timeTaken);
     }
 
+    @Ignore
```

Comment:
Update this test case instead of ignoring it. Make sure that an entry is logged when this happens...

Also - if the test-method:
  timedOperation(Runnable runnable, Class<? extends Exception> expectedExceptionClass)
is no longer used, remove it.